src/components/UserSelector.tsx

```diff
@@ -23,11 +49,24 @@ export const UserSelector: React.FC = () => {
 
       <div className="dropdown-menu" id="dropdown-menu" role="menu">
         <div className="dropdown-content">
-          <a href="#user-1" className="dropdown-item">Leanne Graham</a>
-          <a href="#user-2" className="dropdown-item is-active">Ervin Howell</a>
-          <a href="#user-3" className="dropdown-item">Clementine Bauch</a>
-          <a href="#user-4" className="dropdown-item">Patricia Lebsack</a>
-          <a href="#user-5" className="dropdown-item">Chelsey Dietrich</a>
+          {users.map(user => {
+            const { id } = user;
+
+            return (
+              <a
+                key={id}
+                href={`#user-${id}`}
+                className={classNames(
+                  'dropdown-item',
+                  { 'is-active': selectedUser.id === id },
+                )}
+                onMouseDown={() => selectUser(user)}
+                // onClick={() => setIsChoosing(false)}
```

Comment:
There is a commented-out line of code. If unnecessary, remove the line. If required, uncomment it.